packages/manager/src/features/PlacementGroups/PlacementGroupsAssignLinodesDrawer.tsx

```diff
@@ -0,0 +1,256 @@
+    allPlacementGroups
+  );
+
+  if (
+    !linodes ||
+    !selectedPlacementGroup ||
+    linodesError ||
+    regionsError ||
+    allPlacementGroupsError
+  ) {
+    return (
+      <ErrorState errorText="There was a problem retrieving your placement group. Please try again" />
+    );
+  }
+
+  const getLinodeSelectOptions = (): Linode[] => {
+    return (
+      linodes.filter((linode) => {
+        const isNotAlreadyAssigned = !linodesFromAllPlacementGroups.includes(
+          linode.id as number
+        );
+        const isNotAssignedInDrawer = !localLinodesSelection.find(
+          (l) => l.id === linode.id
+        );
+
+        return isNotAlreadyAssigned && isNotAssignedInDrawer;
+      }) ?? []
+    );
+  };
+
+  const { affinity_type, label } = selectedPlacementGroup;
+  const placementGroupRegion: Region | undefined = regions?.find(
+    (region) => region.id === selectedPlacementGroup.region
+  );
+  const linodeSelectLabel = placementGroupRegion
+    ? `Linodes in ${placementGroupRegion.label} (${placementGroupRegion.id})`
+    : 'Linodes';
+
+  const drawerTitle =
+    label && affinity_type
+      ? `Add Linodes to Placement Group ${label} (${AFFINITY_TYPES[affinity_type]})`
+      : 'Add Linodes to Placement Group';
+
+  const removableSelectionsListLabel = (
+    <>
+      <FormLabel htmlFor="pg-linode-removable-list">
+        {label && affinity_type
+          ? `Linodes Assigned to Placement Group ${label}
+        (${AFFINITY_TYPES[affinity_type]})`
+          : 'Linodes Assigned to Placement Group'}
+      </FormLabel>
+      <Typography component="span" display="block" fontSize="0.8rem">
+        Maximum Number of Linodes for this group:{' '}
+        {selectedPlacementGroup.capacity}
+      </Typography>
+    </>
+  );
+
+  const handleAssignLinode = async (e: React.SyntheticEvent<HTMLElement>) => {
+    e.preventDefault();
+    setSelectedLinode(null);
+
+    if (!selectedLinode) {
+      return;
+    }
+
+    setLocalLinodesSelection([...localLinodesSelection, selectedLinode]);
+
+    const payload: AssignLinodesToPlacementGroupPayload = {
+      linodes: [selectedLinode.id],
+    };
+
+    try {
+      await assignLinodes(payload);
+    } catch (error) {
+      setGeneralError(
+        error?.[0]?.reason
+          ? error[0].reason
+          : 'An error occurred while adding the Linode to the group'
+      );
+    }
+  };
+
+  const handleUnassignLinode = (linode: Linode) => {
+    setLocalLinodesSelection(
+      localLinodesSelection.filter((l) => l.id !== linode.id)
+    );
+
+    const payload: UnassignLinodesFromPlacementGroupPayload = {
+      linodes: [linode.id],
+    };
+
+    try {
+      unassignLinodes(payload);
+    } catch (error) {
+      setGeneralError(
+        error
+          ? error?.[0]?.reason
+          : 'An error occurred while removing the Linode from the group'
+      );
+    }
+  };
+
+  const placementGroupFull = hasPlacementGroupReachedCapacity(
```

Comment:
Since we're prefixing booleans with `is` in this file
```suggestion
  const isPlacementGroupFull = hasPlacementGroupReachedCapacity(
```

Reply:
sure thing